Modules/Loadable/Markups/MRML/vtkMRMLMarkupsNode.cxx

```diff
@@ -138,7 +138,7 @@ void vtkMRMLMarkupsNode::ReadXMLAttributes(const char** atts)
 
   vtkMRMLReadXMLBeginMacro(atts);
   vtkMRMLReadXMLBooleanMacro(locked, Locked);
-  vtkMRMLReadXMLStdStringMacro(markupLabelFormat, MarkupLabelFormat);
```

Comment:
We need to keep this line. It will call `SetMarkupLabelFormat()` which should set ControlPointLabelFormat without logging a warning (because all existing scene files will contain `markupLabelFormat` attribute name). Since the label format is stored in the mkp.json file, too, but we can do it later (if we touch the label formatting for other reasons anyway).

Reply:
I've restored this line here, but restoring a mrk.json doesn't seem to log the warning? I've been trying to replicate that before commenting that warning out.

Reply:
I would expect that the warning is logged if you load a scene that is saved by any earlier Slicer version, because in those scene files the "MarkupLabelFormat" attribute is used.

Reply:
@lassoan I've tried this again but I don't see a warning log when loading of an older scene. Is this because the mrk.json file has a "labelFormat" field and not a "MarkupLabelFormat" field? You can test and see if you can load a scene and have the warning logged from reading it.

Reply:
It is not related to the json file content, but to the scene content (read XML). The warning should appear when a saved scene is loaded.

Reply:
When I load a scene file that includes a mark.json type object it does not log the warning.

Reply:
@lassoan I was able to replicate and have removed the logged warning so that the scene file MarkupLabelFormat attribute doesn't trigger a warning on load.